### DISCUSSION TITLE
how to use pyG GNN explainer to explain graph level prediction and visualize the graph.

### DISCUSSION THREAD

**Discussion by ravi5233** [pyg-team/pytorch_geometric#8141] [Q&A]
I trained the TransformerConv model, i need to check prediction of model with graph level. My dataset is molecular structures and i want to visualize the atom weight heatmap which part of molecular structure responsible for a particular predicted instance. I would like to seek help from pyG community. Thank you for your help.

**Comment by rusty1s** [comment]
Did you take a look at the `return_attention_weights` argument of `TransformerConv`?
